config/ceph-config.service

```diff
@@ -0,0 +1,35 @@
+[Unit]
```

Comment:
Sorry, I think I misread your intent.  It doesn't strike me as necessary to separate the commits:  to me, they seem to be part of the same thought.